README.md

```diff
@@ -158,43 +180,14 @@ func NewBar(a, b Attrib) (*Bar, error) { ... }
 
 An example of an ast can be found [here](example/bools/ast/ast.go)
 
-## Release Notes for gocc 2.1
-
-### Changes
-
-1. no_lexer option added to suppress generation of lexer. See the user guide.
-
-2. Unreachable code removed from generated code.
-
-### Bugs fixed:
-
-1. gocc 2.1 does not support string_lit symbols with the same value as production names of the BNF. E.g. (t2.bnf):
-
-```
-A : "a" | "A" ;
-```
-
-string_lit "A" is not allowed.
-
-Previously gocc silently ignored the conflicting string_lit. Now it generates an ugly panic:
-
-```
-$ gocc t2.bnf
-panic: string_lit "A" conflicts with production name A
-```
-
-This issue will be properly resolved in a future release.
-
 ## Users
 
 These projects use gocc:
 
 * [gogo](https://github.com/shivansh/gogo) - [BNF file](https://github.com/shivansh/gogo/blob/master/src/lang.bnf) - a Go to MIPS compiler written in Go
 * [gonum/gonum](https://github.com/gonum/gonum) - [BNF file](https://github.com/gonum/gonum/blob/master/graph/formats/dot/internal/dot.bnf) - DOT decoder (part of the graph library of Gonum)
-* [llir/llvm](https://github.com/llir/llvm) - [BNF file](https://github.com/llir/llvm/blob/master/asm/internal/ll.bnf) - LLVM IR library in pure Go
```

Comment:
Maybe we can still link to the historical bnf

https://github.com/llir/llvm/blob/28149269dab73cc63915a9c2c6c7b25dbd4db027/asm/internal/ll.bnf

It is quite useful to have more examples when one is starting out